src/common/ga.js

```diff
@@ -0,0 +1,25 @@
+import { waitForGlobal } from './utils';
+
+let client;
+const getClient = () => {
+  if (client) {
+    return Promise.resolve(client);
+  }
+
+  return waitForGlobal('ga').then(gaClient => {
+    client = gaClient;
+    return client;
+  });
+};
+
+export const trackEvent = (category, action, label, value) => {
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
Let's avoid if-blocks wrapping entire functions. It's preferable to early exit.

Reply:
Agree